### DISCUSSION TITLE
Add the ability to delete an alias from Android mobile app

### DISCUSSION THREAD

**Discussion by nopapersplease** [simple-login/app#739] [Feature Requests]
The Android mobile app does not appear to have the ability to delete an alias. I've scoured and searched each screen and I can't find a button on an alias record to delete it. It can only be edited or disabled. The only way that I can find to delete an alias is to log into the desktop web client interface. Please add this feature. For added security, require two-factor authentication code (if enabled) to finalize a delete. Thanks.

**Comment by nguyenkims** [comment]
@ntnhon I believe we can delete an alias on Android right?

**Comment by ntnhon** [comment]
Hello, you can swipe from right to left an alias to reveal the delete option. You're not the first to raise the question so I guess I'll make the delete option more accessible elsewhere. For your information here is a screenshot of the new alias detail page on iOS. From there you can pin/unpin and delete an alias. But you have to wait a little bit before it comes to Android.

![IMG_EF7A7577FFDC-1](https://user-images.githubusercontent.com/4153453/148190501-4627fe35-ecfc-4c16-9cee-51a8eaa984c8.jpeg)

**Reply by nopapersplease** [reply]
I did find the delete option after swiping right to left. I see the reason for the design choice. It does prevent accidental deletes. I just didn't know it was there. It complicates the story a bit but maybe add a configuration option to enable an always-visible delete button or use the swipe action to reveal it.

**Reply by ntnhon** [reply]
I think that we don't delete aliases so often that we really need a visible button that takes up the phone's screen real estate permanently. A swipe to delete and an option in detail page are enough in my opinion. Other than that, you can always deactivate an alias if it becomes too spammy because deletion is a destructive action, it must be used with a thoughtful consideration. You might already know this but you can't recover a deleted alias.

**Reply by nopapersplease** [reply]
I suppose one would be more likely to delete an alias during trial period, but once one starts using the service for real, deletes would be far less common. There's no need to change the app; now that I know where to look I can delete if needed. 

Thanks for the engagement on this topic. I really enjoyed my trial use and I've decided to become a full premium user.

**Reply by ntnhon** [reply]
You're welcome! I'm glad that you decide to become premium user :)
